### DISCUSSION TITLE
Global List of in use ports

### DISCUSSION THREAD

**Discussion by jderagon** [louislam/dockge#667] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

Was wondering if there could be a way to have some sort of global port mapping of all services running, or maybe just an indicator of a local port already in use when editing a compose. When setting up more and more services and writing the docker compose I forget what port everything is running on and have to hunt around to see which container is using which port and thought this might be a nice feature to see where all the services are running at a glance.

### ✔️ Solution

A list somewhere of the stack name and ports in use to see at a glance which ports they are using. 

### ❓ Alternatives

A warning when submitting a compose that uses the port of another service. 

### 📝 Additional Context

Just a minor suggestion here, but thought it would be helpful when setting up new composes and choosing a port for it to run on. Thank you again for the great software!

**Comment by michaelschmalzl** [comment]
Well the Feature would be usefull for Ports only running in Host-Mode.
Has it been implemented yet ?

**Comment by coyodi** [comment]
+1 this would be a great addition. Had the idea after looking at dockpeek.

**Comment by lincooln** [comment]
yes, it would be convenient to have an option to display the ports next to the stack name. usually there is only one port, rarely more. 
or to split the list into two columns, with the name on the left and the ports on the right. This would allow for sorting by name or ports.

**Comment by tismofied** [comment]
I have [portracker](https://github.com/mostafa-wahied/portracker) running for this exact issue. I never remember what port is used in what lol

**Reply by michaelschmalzl** [reply]
That doesn't solve that when you have services running on host mode.
The solution he offered is nice and also running dockpeek now to scan for all services in my LAN.
There you also have the possibility to label the ports and so on.
